src/frontend/Semantic_error.ml

```diff
@@ -322,6 +335,8 @@ module ExpressionError = struct
     | NotPrintable -> Fmt.pf ppf "Functions cannot be printed."
     | EmptyArray ->
         Fmt.pf ppf "Array expressions must contain at least one element."
+    | EmptyTuple ->
+        Fmt.pf ppf "Tuple expressions must contain at least one element."
```

Comment:
The parser also cannot construct an empty array, the error right above this one. It could just be replaced with an `assert false` or something but this is a bit more self-documenting and allows for the potential for something like an empty tuple to exist in the future.